src/components/OrgPostCard/OrgPostCard.test.tsx

```diff
@@ -14,11 +17,26 @@ const MOCKS = [
     },
     result: {
       data: {
-        organizations: [
-          {
-            _id: '1',
-          },
-        ],
+        removePost: {
+          _id: '1',
+        },
+      },
+    },
+  },
+  {
+    request: {
+      query: UPDATE_POST_MUTATION,
```

Comment:
same here

Reply:
This dummy data is to mock graphql queries used in OrgPostCard.tsx file.